firefox-ios/Client/Telemetry/AdsTelemetryHelper.swift

```diff
@@ -6,14 +6,14 @@ import Foundation
 import WebKit
 import Glean
 
-public enum BasicSearchProvider: String {
+enum BasicSearchProvider: String {
```

Comment:
Easy fix, those were all public but they don't need to be. Now it's all implicitly `Sendable`